rfcs/text/0005-host-metric-fields.md

```diff
@@ -26,6 +26,17 @@ Proposed 7 new fields are:
 <!--
 Stage 1: Describe at a high level how this change affects fields. Which fieldsets will be impacted? How many fields overall? Are we primarily adding fields, removing fields, or changing existing fields? The goal here is to understand the fundamental technical implications and likely extent of these changes. ~2-5 sentences.
 -->
+This RFC calls for the addition of host fields to collect basic monitoring metrics from a host or VM such as CPU, network and disk.
+
+| field | type | description |
+| --- | --- | --- |
+| `host.cpu.pct` | scaled_float | Percent CPU used. This value is normalized by the number of CPU cores and it ranges from 0 to 1. |
```

Comment:
ECS guidelines recommend not using abbreviated terms like `pct`.

Reply:
I'd prefer using term `utilization` over `percentage`. I see capturing the CPU % as the way to measure CPU utilization.

Reply:
Or perhaps to maintain the unit in the field name, something like `usage_percentage`.

Reply:
Sounds good! How about changing this field name to `host.cpu.usage_percentage`?

Reply:
Late to the game, but ++ on using _meta for it.

Reply:
Done! Changed to `host.cpu.usage` already 🙂 Thank you for all the input!